src/zenml/config/pipeline_run_configuration.py

```diff
@@ -46,6 +47,7 @@ class PipelineRunConfiguration(
     steps: Dict[str, StepConfigurationUpdate] = {}
     settings: Dict[str, SerializeAsAny[BaseSettings]] = {}
     tags: Optional[List[str]] = None
+    stack: Optional[Stack] = None
```

Comment:
This should be an `Optional[str]` instead. The `Stack` object is not something that can be defined in a yaml file, which is usually how users specify attributes of this run configuration class. The value for `stack` will be the name or ID of a stack, which will then be passed to `temporary_active_stack`.